source/contentprovider.d

```diff
@@ -245,7 +245,7 @@ class ContentProvider
 		settings.flags = MarkdownFlags.backtickCodeBlocks | MarkdownFlags.vanillaMarkdown | MarkdownFlags.tables;
 		settings.urlFilter = (string link, bool) {
 			import std.algorithm.searching : startsWith;
-			if (link.startsWith("http") || link.startsWith("https") || link.startsWith("irc") || link[0] != '/')
+			if (link.startsWith("http") || link.startsWith("irc") || link[0] == '/')
```

Comment:
@wilzbach I don't understand this change (the removal of `link.startsWith("https")`. Can you explain why links starting with `https` should be validated using `isValidLink`. Looking at its implantation, it seems that it can only handle links of the `welcome/welcome-to-d` form, and definitely not links starting with `https`.